src/components/JoinPage/JoinContent.js

```diff
@@ -0,0 +1,139 @@
+import React from "react"
+import styled from "styled-components"
+
+import { lightenDarkenColor, SB_NAVY } from "@colors"
+import SquareLogo from "components/squareLogo"
+import { capitalize } from "utils/string"
+
+const Wrapper = styled.div`
+  border-left: 6px solid ${props => props.color};
+  transition: border-left 0.3s;
+  padding-left: 1em;
+  margin: 2em 0;
+  @media (min-width: 1000px) {
+    padding-left: 1.5em;
+    margin: 4em 8em;
+  }
+`
+
+const ContentHeaderWrapper = styled.h3`
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.8em;
+  margin-bottom: 0;
+`
+
+const HeaderText = styled.div`
+  font-weight: 500;
+`
+
+const ContentBody = styled.div`
```

Comment:
I think this should just use the styling from styles/Body. As is, the line-height looks inconsistent with the rest of the site

Reply:
Wow, didn't see that. Will do.